src/bot/exts/dragonfly/dragonfly.py

```diff
@@ -131,7 +131,7 @@ async def report(self: Self, interaction: discord.Interaction, button: discord.u
 def _build_package_scan_result_embed(scan_result: PackageScanResult) -> discord.Embed:
     """Build the embed that shows the results of a package scan."""
     embed = discord.Embed(
-        title=f"New Scan Result: {scan_result.name} v{scan_result.version}",
+        title=f"Malicious package found: {scan_result.name} @ {scan_result.version}",
         description=f"```YARA rules matched: {', '.join(scan_result.rules) or 'None'}```",
```

Comment:
Is this capable of being `None` here?
If it's hit the threshold, it has to have rules, right?

Reply:
Yeah, but we can't really specify that in typing. Unless we have a separate type for that or something along those lines. Figured it'd just be easier to do this. But yeah, you're right in that we should never see this (unless the database was changed manually or something like that)